src/app/sidenav/sidenav.component.ts

```diff
@@ -8,6 +8,8 @@ import {MatSidenav} from '@angular/material';
 })
 export class SidenavComponent implements OnInit {
   @ViewChild('sidenav') sidenav: MatSidenav;
+  communities: Array<any>;
```

Comment:
You can move the array initialization here so it'll be `communities: Array<any> = []`